wp-includes/widgets/class-wp-widget-audio.php

```diff
@@ -23,9 +23,10 @@ class WP_Widget_Audio extends WP_Widget_Media {
 	 * @access public
 	 */
 	public function __construct() {
-		parent::__construct( 'audio', __( 'Audio' ), array(
-			'classname'   => 'widget_audio',
+		parent::__construct( 'wp-audio', __( 'Audio' ), array(
```

Comment:
Thanks for the link to the [Slack conversation](https://wordpress.slack.com/archives/core-customize/p1489080794278492). It's not precedented to prefix widget id bases with `wp-` in core. Existing ID bases have `text`, `meta`, `search`, and so on. What about using `media_` instead as a prefix? This would seem more in line with existing core conventions and it would reflect that these are all subclasses of the one media widget.

Alternatively, as was done with term meta, existing plugins be updated to be compatible with the new widget as opposed to accommodating core for plugins that happen to use that ID base. This would be more difficult than using a prefix, however. I like `media_*`.